linkedin.txt

```diff
@@ -11,3 +11,17 @@ www.linkedin.com##.premium-upsell-link--extra-long
 ! Hide LinkedIn messages overlay
 www.linkedin.com##.msg-overlay-container-reflow.msg-overlay-container
 www.linkedin.com##.mb2.artdeco-card.community-panel
+
+! Hide LinkedIn Pay for premium under the profile in Feed page
+www.linkedin.com##.text-align-left.link-without-hover-state.t-bold.t-black.t-12.feed-identity-module__anchored-widget--premium-upsell.feed-identity-module__anchored-widget.link-without-visited-state.app-aware-link
+
+! Hide LinkedIn  "Try Premium for free" top bar
+www.linkedin.com##.premium-upsell-link--truncate.global-nav__spotlight-upsell.global-nav__primary-link--premium.global-nav__primary-link.link-without-visited-state
```

Comment:
```suggestion
```
I think these can be removed. But double check please.

Reply:
Hey, I double-checked that for me, they're visible, so I'll just add them at my filters (also w/ Firefox)

Reply:
Feel free to keep them, or open a new PR with them and I will test again later.